app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,42 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        From the home screen, swipe down to display the favorites panel and swipe up to hide it.
+        Swipe 🡇 from the home screen to show all apps, and 🡇 to go back.
         \n
-        \nIf you swipe up when the favorites panel is hidden, it will display the complete list of applications.
-        Swipe down from the top of this list to come back to the home screen.
+        \nSwipe 🡇 from the home screen to show favorites, and 🡅 to go back.
         \n
-        \nPress the button below the favorites panel to display the menu of the launcher.
+        \nClick 🔧 when showing favorites to open the menu.
         \n

```

Comment:
I am not a fan of mixing words and emojis, for me the previous proposal with only words was better.

Also, using the latest Firefox I only see white squares for the emojis representing swipes directions.
If they are already not working on GitHub, I would prefer not to use them on phones.

Reply:
The problem is denoting what it is based on position, because in some sense words is the only thing relating to the pictogram in the menu.

Some launchers use the gear icon for both, and some have a white gear icon for one thing, and a black for system settings, but that I can't get used to.

Reply:
Thank you for the correction, yes it is ok for me to use both like you did for the menu button.

Just noticed that an error introduced itself on line 8 with the up/down directions for the app drawer, I will correct it.

Reply:
I changed the order of the events to segway easier into the menu button explanation.